app/components/listing/ListingTitleLink.tsx

```diff
@@ -23,6 +23,10 @@ export const ListingTitleLink = ({
       <OrganizationCard org={listing} />
     );
 
+  // Don't translate names of organizations
+  const classNames = isService ? "popover" : "popover weglot-no-translate";
+  const translate = isService ? undefined : "no";
```

Comment:
That is a good question... I will ask but I could see it going either way. It's a tough call though. The reason is because a lot of them seem to pertain to certain phrases (for example, ALRP's Legal Help as "Ayuda Legal": https://sfserviceguide.org/services/2855). Some are specific program names, though (e.g. San Francisco YouthWorks), which makes them less than ideal to translate.

Frankly it might have to be a case by case basis, in the ideal world.

But yeah, SFSG actually does translate the service name on the ServiceListingPage as seen by the ALRP service page above. Is that a bug, then?

Reply:
Yep, that is a bug on the SFSG side -- we should at least be consistent. I will Slack the product-design channel to gauge people's thoughts. I could see it going either way. Let us know what Our 415 says. I think we could, probably, follow their lead.

Reply:
It sounds like SFSG wants to avoid translating service names at this point. I will go with that in this PR, and confirm with Our415 that that's all right (haven't gotten a chance yet).

Reply:
On second thought, I'll confirm actually we have a chance to sync with Our415 on Tues. It may be that we'll need to specify `translateOrgNames` / `translateServiceNames` in the whitelabel options. That's because the Search Results on Our415 currently do actually translate the names: https://www.our415.org/v2-search/?category=287&categoryName=Youth%20Resources

Reply:
Thanks Edward, that sounds good to me. It seems like we don't want to translate service names on SFSG for now, but I added an agenda item to discuss during tonight's product meeting and will bring up Our 415. Let us know what they say if you get a hold of them on Tuesday. I'd be curious to hear their reasoning too, if you all get into that.

Reply:
I've asked them, but I haven't been able to get their opinion on it yet. Will proceed with the above plan, and try to convert to `.notranslate` in a separate commit.